yotiprofile.go

```diff
@@ -0,0 +1,146 @@
+package yoti
+
+import (
+	"github.com/getyoti/yoti-go-sdk/attribute"
+)
+
+const (
+	attrConstSelfie                  = "selfie"
+	attrConstGivenNames              = "given_names"
+	attrConstFamilyName              = "family_name"
+	attrConstFullName                = "full_name"
+	attrConstMobileNumber            = "phone_number"
+	attrConstEmailAddress            = "email_address"
+	attrConstDateOfBirth             = "date_of_birth"
+	attrConstAddress                 = "postal_address"
+	attrConstStructuredPostalAddress = "structured_postal_address"
+	attrConstGender                  = "gender"
+	attrConstNationality             = "nationality"
```

Comment:
These are part of the core-api... not sure redefining them is worth it/a good idea...

Reply:
We have had to redefine them as we want anyone to be able to check out the SDK and build it, so we can't include private APIs